pkg/graph/resolvers/trb_request.go

```diff
@@ -0,0 +1,76 @@
+package resolvers
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/storage"
+)
+
+//TRBRequestCreate makes a new TRB request
+func TRBRequestCreate(ctx context.Context, requestType models.TRBRequestType, store *storage.Store) (*models.TRBRequest, error) {
+	princ := appcontext.Principal(ctx)
+	trb := models.NewTRBRequest(princ.ID())
+	trb.Type = requestType
+	trb.Status = models.TRBSOpen
+	//TODO make sure this is wired up appropriately
+
+	createdTRB, err := store.TRBRequestCreate(appcontext.ZLogger(ctx), trb)
+	if err != nil {
+		return nil, err
+	}
+
+	//TODO create place holders for the rest of the related sections with calls to their stores
```

Comment:
Is there a Jira ticket for writing the placeholders? Or will we just add them as part of other TRB-related tickets that add more fields to `models.TRBRequest`?

Reply:
this will be part of the other ticket sections.